internal/zinc-apiinfo/src/main/scala/sbt/internal/inc/ClassToAPI.scala

```diff
@@ -384,7 +384,7 @@ object ClassToAPI {
   ): api.Def = {
     val varArgPosition = if (varArgs) paramTypes.length - 1 else -1
     val isVarArg = List.tabulate(paramTypes.length)(_ == varArgPosition)
-    val pa = (paramAnnots, paramTypes, isVarArg).zipped map {
+    val pa = paramAnnots.lazyZip(paramTypes).lazyZip(isVarArg).map {
```

Comment:
```
[warn] -- Deprecation Warning: /home/runner/work/zinc/zinc/internal/zinc-apiinfo/src/main/scala/sbt/internal/inc/ClassToAPI.scala:387:49 
[warn] 387 |    val pa = (paramAnnots, paramTypes, isVarArg).zipped map {
[warn]     |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[warn]     |method zipped in class Ops is deprecated since 2.13.0: Use xs.lazyZip(ys).lazyZip(zs)
```